### PR TITLE
Update documentation to warn users not to store sprites and text in a struct

### PR DESCRIPTION
**Pull request by dogunbound**

### CODE REVIEW

File: src/graphics/sprite.rs [hunk 1]
```diff
@@ -12,6 +12,11 @@ use std::{
     ptr::{self, NonNull},
 };
 
+/// !! WARNING !!
```
Comment (crumblingstatue) [inline src/graphics/sprite.rs:15]: I wouldn't put such a dramatic warning here.

Maybe `Note:` would be better.

File: src/graphics/sprite.rs [hunk 2]
```diff
@@ -12,6 +12,11 @@ use std::{
...
+/// Currently there is no way to store sprites in a struct. All sprites must be made right before
```
Comment (crumblingstatue) [inline src/graphics/sprite.rs:16]: It's not impossible to store them in a struct, it's just not very practical.
I would rephrase this as
> Currently, it is not feasible to store sprites long term.
> A common pattern with rust-sfml is to create a `Sprite` right before you start drawing, and draw all the sprites you want with it. You can change its properties using `set_texture`, `set_position`, etc., before drawing it, as many times as you need to.

File: src/graphics/sprite.rs (hunk 1 above)
Comment (dogunbound) [reply src/graphics/sprite.rs]: Changed to Note:

File: src/graphics/sprite.rs (hunk 2 above)
Comment (dogunbound) [reply src/graphics/sprite.rs]: I copy and pasted that for sprite.rs

text.rs you may want to read.
